### ISSUE TITLE
Intake Import Error in py2

### ISSUE THREAD

**Issue by patanijo** [ContinuumIO/intake#166]
I have tried several versions and tried installing in root as well as in its own conda environment, but I get the following error:

```python
>>> import intake
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/johnpatanian/anaconda2/lib/python2.7/site-packages/intake/__init__.py", line 19, in <module>
    from .source import csv, textfiles, npy
  File "/Users/johnpatanian/anaconda2/lib/python2.7/site-packages/intake/source/csv.py", line 111
    urlpath, *_ = self._get_cache(self._urlpath)
             ^
SyntaxError: invalid syntax
>>> exit()
C02QF6EQG8WP:Downloads johnpatanian$ conda list intake
```
Running on Python 2.7

**Comment by patanijo** [comment]
Thank you.  I will try again in a Python 3 environment. In addition I'll be happy to take a look to see if I can identify areas where there are python 2 incompatibilities.

**Comment by martindurant** [comment]
Thanks, @patanijo . Let's leave this issue open in the meantime, fo other people are more likely to find it.

**Comment by patanijo** [comment]
It looks like the python 2.7 compatibility is limited to three places, and it is all the same compatibility issue with starred expression in the return value from a function call:

**In csv.py**
```python
    def _get_schema(self):
        urlpath, *_ = self._get_cache(self._urlpath)
```
** In np.py **
``` python
    def _get_schema(self):
        from dask.bytes import open_files
        import dask.array as da
        if self._arr is None:
            path, *_ = self._get_cache(self.path)
```
** IN textfiles.py **
``` python
    def _get_schema(self):
        from dask.bytes import open_files
        if self._files is None:

            urlpath, *_ = self._get_cache(self._urlpath)
```
Will try and nvestigate to see if there are alternatives that don't break existing tests.

**Comment by martindurant** [comment]
I imagine there may be more subtle cases too. For this pattern, you can always do

```
path, *_ = self._get_cache(self.path)
->
path = self._get_cache(self.path)[0]
```

**Comment by martindurant** [comment]
Fixed in #192

**Comment by patanijo** [comment]
@martindurant Thank you! I will let you know if I find any other edge cases.
